### ISSUE TITLE
Let user set language during setup

### ISSUE THREAD

**Issue by jniggemann** [alex-LE/yourTinyTodo#23]
If we manage to release with more language files included, the user could choose their language during setup.

**Comment by alex-LE** [comment]
good idea, but we should not include too much languages in the main release, because we have to manage to much file for every change, or we have to wait with a release until all stuff is translated

**Comment by jniggemann** [comment]
You're right, that would delay releases... Perhaps we could put the language files on our webspace, the setup could then download the chosen language...
